src/libs/TransactionUtils.js

```diff
@@ -141,6 +141,11 @@ function getUpdatedTransaction(transaction, transactionChanges, isFromExpenseRep
         shouldStopSmartscan = true;
     }
 
+    if (_.has(transactionChanges, 'category')) {
+        updatedTransaction.modifiedCategory = transactionChanges.category;
+        shouldStopSmartscan = true;
```

Comment:
@puneetlath, @mountiny, guys, could you please clarify this value - `shouldStopSmartscan`. I set `true`, because all other have the same, but I am not sure about a category.

Reply:
It should not stop SmartScan.